.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+
+branches:
+  only:
+    - gh-pages
+
+env:
+  global:
+    - URL="https://w3c.github.io/mediacapture-record/W3CTRMANIFEST"
+    - DECISION="https://lists.w3.org/Archives/Public/public-media-capture/@@@"
```

Comment:
oops.
2 archived emails that relates to automatic publication of the Media Capture TF docs:
https://lists.w3.org/Archives/Public/public-media-capture/2015Dec/0031.html
https://lists.w3.org/Archives/Public/public-webrtc-editors/2016Feb/0024.html

I'll use the first one for the DECISION URI.